pkg/trace/obfuscate/sql_test.go

```diff
@@ -264,10 +264,18 @@ func TestSQLQuantizeTableNames(t *testing.T) {
 				"REPLACE INTO sales_2019_07_01 (`itemID`, `date`, `qty`, `price`) VALUES ((SELECT itemID FROM item1001 WHERE `sku` = [sku]), CURDATE(), [qty], 0.00)",
 				"REPLACE INTO sales_?_?_? ( itemID, date, qty, price ) VALUES ( ( SELECT itemID FROM item? WHERE sku = [ sku ] ), CURDATE ( ), [ qty ], ? )",
 			},
+			{
+				"SELECT ddh19.name, ddt.tags FROM dd91219.host ddh19, dd21916.host_tags ddt WHERE ddh19.id = ddt.host_id AND ddh19.org_id = 2 AND ddh19.name = 'datadog'",
```

Comment:
Can we put numbers in some of the column names to clarify that we expect numbers to be obfuscated there as well?

Reply:
Added: https://github.com/DataDog/datadog-agent/pull/8860/commits/4eaba64a8fafaa0f9d2ce9a0d5fad791a203406b

Reply:
@alexbarksdale the test cases here don't give me a lot of confidence in this feature because there are potentially a large number of edge cases. Can you work on expanding the coverage of test cases?

Reply:
@justiniso Added a medium and large complex query, if there's a specific statement you're looking to see let me know, I'm more than happy to add another test case. Although I think the large complex query may be enough to form an opinion on this change(?). The large complex query added is a bit difficult to read in the test so I added it to the body of the PR for ease. https://github.com/DataDog/datadog-agent/pull/8860/commits/d7e9df9d3698ba607fa960cd3e7d440d347dcfe7